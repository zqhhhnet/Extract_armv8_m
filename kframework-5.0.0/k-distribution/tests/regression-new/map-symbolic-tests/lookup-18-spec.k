// Copyright (c) 2019 K Team. All Rights Reserved.

requires "map-tests.k"

module LOOKUP-18-SPEC
    imports MAP-TESTS

    rule <k> lookup ( (X:MyId |-> 1 MAP:Map Y:MyId |-> 2) [ Y:MyId ] ) => . </k> requires X =/=K Y

endmodule
