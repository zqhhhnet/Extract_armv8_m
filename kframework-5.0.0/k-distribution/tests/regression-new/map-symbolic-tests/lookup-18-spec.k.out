<k>
  lookupResult ( 2 )
</k>
